recaf-assembler/src/main/java/me/coley/recaf/assemble/pipeline/AssemblerPipeline.java

```diff
@@ -259,6 +260,10 @@ public Unit getUnit() {
 		return unit;
 	}
 
+	public List<Group> getLatestAst() {
```

Comment:
Will look over usage later, but having this being _"latest ast"_ may be confusing when comparing to _"unit"_ which is also occasionally referred to as the AST. Could make the naming convention of Jasm AST more specific to differentiate from Recaf AST?

Reply:
i thought about Parser AST